src/cli/speed.cpp

```diff
@@ -319,79 +323,78 @@ class Speed final : public Command {
          points of the most interesting or widely used algorithms.
          */
 
-         return {
```

Comment:
Maybe add a `clang-format off` annotation here since I don't think the reformat improves readability in the first place, and makes the diff noisy.